src/main/java/school/faang/user_service/service/UserService.java

```diff
@@ -0,0 +1,23 @@
+package school.faang.user_service.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import school.faang.user_service.dto.SkillDto;
+import school.faang.user_service.entity.Skill;
+import school.faang.user_service.entity.User;
+import school.faang.user_service.repository.UserRepository;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class UserService {
```

Comment:
пропиши функциональность в тестах, чтобы не было 
java.lang.NullPointerException: Cannot invoke "school.faang.user_service.service.UserService.extracted(school.faang.user_service.dto.SkillDto, school.faang.user_service.entity.Skill)" because "this.userService" is null
так же обрати внимание на то, что метод в UserService не принимает ничего на прямую связанного с той сущностью, с которой он намеревается работать, твоё решение крайне не гибкое из-за SkillDto, если ты будешь передавать просто тот тип, который принимает метод userRepository.findAllById, то будешь отвязан от SkillDto и этот метод будет проще переиспользовать в других местах сервиса
так же называй методы так, чтобы было понятно что он делает название extracted не говорит буквально не о чем, лучше назвать его saveSkillFromAllUsers

Reply:
хорошо))